include/mbgl/ios/MGLAccountManager.h

```diff
@@ -26,12 +26,12 @@ NS_ASSUME_NONNULL_BEGIN
  */
 + (nullable NSString *)accessToken;
 
-#pragma mark Providing User Metrics Opt-Out
+#pragma mark Providing User Telemetry Opt-Out
 
 /**
- Whether in-app user metrics opt-out is configured. If set to the default value of `NO`, a user opt-out preference is expected in a `Settings.bundle` that shows in the application's section within the system Settings app.
- */
-+ (BOOL)mapboxMetricsEnabledSettingShownInApp;
+ Whether in-app user telemetry opt-out is configured. If set to the default value of `NO`, a user opt-out preference was expected in a `Settings.bundle` that shows in the application's section within the system Settings app. Telemetry settings were added directly to the in-app ℹ️ menu, obseleting this method.
```

Comment:
Jazzy doesn’t recognize the deprecated attribute yet. Our convention from the appledoc days is to omit documentation for deprecated and unavailable methods, but if this information is still valuable, I’d stick the last sentence in a `@note`.